stylesheets/_utility.utilities.scss

```diff
@@ -328,3 +323,13 @@ div.no-results {
 .u-width-auto {
     width: auto !important;
 }
+
+%u-no-margin-top,
+.u-no-margin-top {
```

Comment:
Wondering we should name `u-margin-top-none` for consistency with other utility classes.